### DISCUSSION TITLE
Pause and restart db

### DISCUSSION THREAD

**Discussion by pinapoluMahesh** [supabase/supabase#23130] [Questions] [answered]
Initialy i used to have a free plan and db storage is upto 89% and it stopped - nothing is working even apis , sql editor 
so i wanted to pause and restart it .
but after pausing , it has been 24 hours and still it is Pausing.
so i purchased a pro plan hoping that it will work,
but after purchasing also it is still in pausing state.
i tried for contact support also but they are responding way too late 
can anyone help me with the issue ?
![Screenshot from 2024-04-22 11-08-54](https://github.com/supabase/supabase/assets/125038022/8d87db96-497a-4fdf-9339-387a94a938d8)

**Comment by GaryAustin1** [comment, marked as answer]
Support is the only one who can restore it.  I assume you hard-refreshed your browser?
